lagom-cloud-object-storage-example/account-impl/src/main/java/com/lightbend/lagom/account/impl/readside/Storage.java

```diff
@@ -0,0 +1,74 @@
+package com.lightbend.lagom.account.impl.readside;
+
+import akka.Done;
+import akka.actor.ActorSystem;
+import akka.http.javadsl.model.ContentType;
+import akka.http.javadsl.model.ContentTypes;
+import akka.stream.ActorMaterializer;
+import akka.stream.alpakka.s3.MemoryBufferType;
+import akka.stream.alpakka.s3.Proxy;
+import akka.stream.alpakka.s3.S3Settings;
+import akka.stream.alpakka.s3.auth.AWSCredentials;
+import akka.stream.alpakka.s3.auth.BasicCredentials;
+import akka.stream.alpakka.s3.javadsl.MultipartUploadResult;
+import akka.stream.alpakka.s3.javadsl.S3Client;
+import akka.stream.impl.ActorMaterializerImpl;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import akka.util.ByteString;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.Option;
+import scala.Some;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletionStage;
+
+public class Storage {
+
+  private final S3Client client;
+  private final String bucket;
+  private final ActorMaterializer materializer;
+  private Logger logger = LoggerFactory.getLogger(getClass());
+
+  @Inject
+  public Storage(ActorSystem actorSystem) {
+
+    Config config = ConfigFactory.load().getConfig("cloud-object-storage");
+
+    AWSCredentials credentials =
+            new BasicCredentials(
+                    config.getString("credentials.access-key-id"),
+                    config.getString("credentials.secret-access-key")
+                    );
+
+    String host = config.getString("host");
+    Option<Proxy> proxy = Some.apply(new Proxy(host, 443, "https"));
```

Comment:
Should this default be loaded from a `cloud-storage-reference.conf'?

Reply:
We can't have defaults for this values. It's very specific to user's account.

Reply:
do you think `port` and `scheme` cold be reused?

Are the `host` names in COS very user-specific? I know they are in S3, but I'm not familiar with COS.

Reply:
yes, there are a couple to choose from depending on the region for instance. It may be vary from account to account as well, but I'm not sure about it.